ls1intum/Artemis
# `General`: Add text search to markdown and online code editors

### Checklist
#### General


#### Client

### Motivation and Context
Regular Ctrl-F search in the browser does not capture text in the ace editor window.
Searching in the text is helpful, though, e.g. when looking for a specific typo in the problem statement or in the online code editor.

### Description
Enables the built-in search functionality of the ace editor by adding the import for it.
Applies some custom styling to the component to make it look coherent to the general application around it.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a new programming exercise.
2. When editing the problem statement, press Ctrl/Command-F to bring up the search.
3. Finish creating the exercise.
4. Go to ‘Edit in Editor‘, open a file. Check that the same key combination works there as well.


### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
n/a

### Screenshots
![image](https://github.com/ls1intum/Artemis/assets/64250573/a5261926-f1f1-4dd2-8905-4fa38bc0a56d)

![image](https://github.com/ls1intum/Artemis/assets/64250573/4e70655b-df77-4d16-88ac-c10db1eefe8f)

![image](https://github.com/ls1intum/Artemis/assets/64250573/ff767d4e-b125-4b36-b15d-a1b3621e03b8)

![image](https://github.com/ls1intum/Artemis/assets/64250573/e3b44dec-d3ee-498d-9c48-ade03a266883)
